app/screens/Library/components/Book/index.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, Text, Image } from 'react-native';
+
+import defaultImg from '../../../../../assets/no_image.jpeg';
+
+import { styles } from './styles';
+
+function Book({ title, author, img }) {
+  return (
+    <View style={[styles.bookContainer, styles.shadowBox]} >      
```

Comment:
can you add the `shadowBox` styles to `bookContainer`? is not really necessary to split these styles in 2 objects.